database/types.go

```diff
@@ -22,10 +22,12 @@ func NewNullString(s string) sql.NullString {
 	}
 }
 
+// NewNullTime returns a sql.NullTime with the given time.Time. If the time is
+// the zero value, the NullTime is invalid.
 func NewNullTime(t time.Time) sql.NullTime {
```

Comment:
tests to verify this behaviour would be helpful

Reply:
great suggestion, added